content/api/cypress-api/require.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Cypress.require
+---
+
+`Cypress.require` enables utilizing dependencies within the
+[`cy.origin()`](/api/commands/origin) callback function. It is used to require
+modules such as [npm](https://www.npmjs.com/) packages and other local files.
+
+It is functionally the same as using
+[CommonJS `require()`](https://nodejs.org/en/knowledge/getting-started/what-is-require/)
+in browser-targeted code.
+
+## Syntax
+
+```js
+Cypress.require(moduleId)
+```
+
+## Usage
+
+**<Icon name="check-circle" color="green"></Icon> Correct Usage**
+
+```js
+cy.origin('somesite.com', () => {
+  const _ = Cypress.require('lodash')
+  const utils = Cypress.require('./utils')
+
+  // ... use lodash and utils ...
+})
+```
+
+**<Icon name="exclamation-triangle" color="red"></Icon> Incorrect Usage**
+
+```js
+// `Cypress.require()` cannot be used outside the `cy.origin()` callback.
+// Use `require()` instead
+const _ = Cypress.require('lodash')
+
+cy.origin('somesite.com', () => {
+  // `require()` cannot be used inside the `cy.origin()` callback.
+  // Use `Cypress.require()` instead
+  const _ = require('lodash')
+})
+
+// the callback must be inline and cannot be assigned to a variable for
+// `Cypress.require()` to work
+const callback = () => {
+  const _ = Cypress.require('lodash')
+})
+
+cy.origin('somesite.com', callback)
+```
+
+## Examples
+
+See [`cy.origin()` Dependencies / Sharing Code]() for example usages.
```

Comment:
```suggestion
See [`cy.origin()` Dependencies / Sharing Code](/api/commands/origin#Serialization) for example usages.
```

Reply:
Good catch! Fixed in https://github.com/cypress-io/cypress-documentation/pull/4747/commits/5653543ab352b0a62ce3a66039c5da3209c29123